docs/en/advanced-features/logging-setup.md

```diff
@@ -1,3 +1,42 @@
 # Logging Setup 
 
-Currently, this part has not been done yet.
\ No newline at end of file
+Logging Setup is used to integrate the Go Agent with the logging system in the current service. 
+It currently supports the recognition of `Logrus` and `Zap` frameworks. If neither of these frameworks is present, it would output logs using `Std Error`.
+
+You can learn about the configuration details through the "log" configuration item in the [default settings](../../../tools/go-agent/config/agent.default.yaml).
+
+## Agent with Logging system
+
+The integration of the Agent with logs can be divided into the following:
+
+1. **Integrating Agent logs into the Service**: Integrating the logs from the Agent into the framework used by the service.
+2. **Integrating Tracing information into the Service**: Integrating the information from Tracing into the service logs.
+
+### Agent logs into the Service
+
+Agent logs output the current running status of the Agent system, most of which are execution exceptions. 
+For example, communication anomalies between the Agent and the backend service, plugin execution exceptions, etc.
+
+Integrating Agent logs into the service's logging system can effectively help users quickly troubleshoot whether there are issues with the current Agent execution.
+
+### Tracing information into the Service
+
+The Agent would also enhance the existing logging system. 
+When the service outputs log, if the current goroutine contains Tracing data, it would be outputted together with the current logs. 
```

Comment:
This reminds me, whether go agent has added the mechanism that would NOT generate any trace information if the backend is not connected.
This is important in the product env. Once the backend(OAP or any others) is down, there is no point to keep tracing.

Reply:
Nope. For now, the gRPC will keep tracing if the connection is closed. I could create other PR for this.